dsgrid/config/dimensions.py

```diff
@@ -494,6 +502,9 @@ def check_times(cls, ranges, values):
             return ranges
         return _check_time_ranges(ranges, values["str_format"], timedelta(days=365))
 
+    def does_geography_require_time_zone(self):
+        return False
```

Comment:
May be unrelated here, what will we ever use TimeZone.NONE for?

Reply:
I don't know.